tasks/release.py

```diff
@@ -102,22 +101,44 @@ def update_changelog(ctx, new_version):
     if new_version_int[0] == 7:
         v6_tag = _find_v6_tag(ctx, new_version)
         if v6_tag:
-            ctx.run("sed {0} -E 's#^{1}#{1} / {2}#' /tmp/new_changelog.rst".format(sed_i_arg, new_version, v6_tag))
+            ctx.run(
+                "sed {0} -E 's#^{1}#{1} / {2}#' /tmp/new_{3}.rst".format(
+                    sed_i_arg, new_version, v6_tag, target_filename
+                )
+            )
     # remove the old header from the existing changelog
-    ctx.run("sed {0} -e '1,4d' CHANGELOG.rst".format(sed_i_arg))
+    ctx.run("sed {0} -e '1,4d' {1}.rst".format(sed_i_arg, target_filename))
 
-    # merging to CHANGELOG.rst
-    ctx.run("cat CHANGELOG.rst >> /tmp/new_changelog.rst && mv /tmp/new_changelog.rst CHANGELOG.rst")
+    # merging to {target_filename}.rst
+    ctx.run("cat {0}.rst >> /tmp/new_{0}.rst && mv /tmp/new_{0}.rst {0}.rst".format(target_filename))
 
-    # commit new CHANGELOG
+    # commit new {target_filename}
     ctx.run(
-        "git add CHANGELOG.rst \
-            && git commit -m \"Update CHANGELOG for {}\"".format(
-            new_version
+        "git add {0}.rst \
+            && git commit -m \"Update {0} for {1}\"".format(
+            target_filename, new_version
         )
     )
 
 
+@task
+def update_changelog(ctx, new_version):
+    """
+    Quick task to generate the new CHANGELOG using reno when releasing a minor
+    version (linux/macOS only).
+    """
+    _update_changelog(ctx, new_version, "releasenotes", "CHANGELOG")
+
+
+@task
+def update_install_changelog(ctx, new_version):
```

Comment:
How should this task be run? I am having trouble understanding how we would go about it.
If we were to create the changelog, for, say, 1.0.4, what commands would one have to run?

Reply:
Similar to update_changelog:
`inv release.update-install-changelog 1.0.4`

I will update https://github.com/DataDog/mars-jenkins-scripts/blob/master/release_documentation/final_artifacts.md#update-the-changelog-release-coordinator with the info.

Reply:
Similar to `update_changelog`:
`inv release.update-install-changelog 1.0.4`

Reply:
> Similar to `update_changelog`:
> `inv release.update-install-changelog 1.0.4`

That doesn't work for me though, you need to have a tag (I assume `1.0.4`) and if you do the task fails:

```
1.0.4
fatal: ambiguous argument '1.0.0-devel...remotes/origin/1.-1.x': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
loaded configuration file ./releasenotes/config.yaml
ignoring unknown configuration value 'no_show_source' = True
scanning ./releasenotes/notes (branch=*current* earliest_version=1.0.0-devel collapse_pre_releases=True)
Traceback (most recent call last):
  File "/usr/local/bin/reno", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/reno/main.py", line 203, in main
    return args.func(args, conf)
  File "/usr/local/lib/python3.8/site-packages/reno/report.py", line 21, in report_cmd
    ldr = loader.Loader(conf)
  File "/usr/local/lib/python3.8/site-packages/reno/loader.py", line 61, in __init__
    self._load_data()
  File "/usr/local/lib/python3.8/site-packages/reno/loader.py", line 82, in _load_data
    self._scanner_output = self._scanner.get_notes_by_version()
  File "/usr/local/lib/python3.8/site-packages/reno/scanner.py", line 954, in get_notes_by_version
    raise ValueError(
ValueError: earliest-version set to unknown revision '1.0.0-devel'
```

Reply:
Can you try 
`inv release.update-install-changelog 7.24..0`

The version has to be a git label in the repro.

Reply:
Oh, so the tag must be on Github, sorry. Okay, one last question. This
https://github.com/DataDog/datadog-agent/blob/de7193d78dfd15f9feedfe96ed19f6bbddcc7a7b/releasenotes-install/config.yaml#L4
makes me think that the install script releases would be tied to the Agent releases, right? Is that what we want? If so this looks good to me but I am not sure about that.

Reply:
There's some additional work needed. I'll update the PR.

Reply:
There are even more stuff going on, closing this PR for now.